src/virtual_renderer_test.js

```diff
@@ -395,7 +395,7 @@ module.exports = {
         editor.renderer.$loop._flush();
         assert.equal(editor.renderer.content.textContent, "abcdefThis is a long test text that is longer than ");
 
-        assert.equal(editor.session.lineWidgets[0].el.innerHTML, "<div class=\"ghost_text_line_wrapped\">30 characters</div><div></div><div>Ghost3</div>");
+        assert.equal(editor.session.lineWidgets[0].el.innerHTML, `<div class="ghost_text_line_wrapped">30 characters</div><div class=""></div><div class="">Ghost3</div>`);
```

Comment:
Where do the empty classes come from? Are they a consequence of using `dom.createElement`?

Reply:
Could you confirm whether this was tested manually (e.g., in the kitchen sink) too?

Reply:
the empty classes are a consequence from ` chunkDiv.className = el.wrapped ? "ghost_text_line_wrapped": "";`. I can change it so we only set the classname if it's needed that might be cleaner indeed.

Reply:
> Could you confirm whether this was tested manually (e.g., in the kitchen sink) too?

yes 

<img width="1435" alt="Screenshot 2024-06-27 at 11 22 21" src="https://github.com/ajaxorg/ace/assets/34573235/8ab89a32-3932-41ea-aefa-0db009fe279b">